configdefinitions/src/vespa/attributes.def

```diff
@@ -7,6 +7,7 @@ attribute[].collectiontype      enum { SINGLE, ARRAY, WEIGHTEDSET } default=SING
 # Deprecated/ do-not-use, will soon be GCed.
 attribute[].dictionary.ordering enum { ORDERED, UNORDERED } default = ORDERED
 attribute[].dictionary.type     enum { BTREE, HASH, BTREE_AND_HASH } default = BTREE
+attribute[].dictionary.match    enum { CASE_SENSITIVE, CASE_INSENSITIVE } default=CASE_INSENSITIVE
```

Comment:
@baldersheim consider changing the enums to CASED and UNCASED.

Reply:
Fixed in #17314